csunplugged/utils/BaseLoader.py

```diff
@@ -36,29 +44,48 @@ def setup_md_to_html_converter(self):
         """
         templates = self.load_template_files()
         extensions = [
-            'markdown.extensions.fenced_code',
-            'markdown.extensions.codehilite',
-            'markdown.extensions.sane_lists',
-            'markdown.extensions.tables',
+            "markdown.extensions.fenced_code",
+            "markdown.extensions.codehilite",
+            "markdown.extensions.sane_lists",
+            "markdown.extensions.tables",
             mdx_math.MathExtension(enable_dollar_delimiter=True)
         ]
         self.converter = Verto(html_templates=templates, extensions=extensions)
         custom_processors = self.converter.processor_defaults()
-        custom_processors.add('remove-title')
+        custom_processors.add("video")
+        custom_processors.add("remove-title")
         self.converter.update_processors(custom_processors)
 
-    def convert_md_file(self, md_file_path):
-        """Convert the given Markdown file to HTML.
+    def convert_md_file(self, md_file_path, config_file_path, heading_required=True):
+        """Return sthe Verto object for a given Markdown file.
 
         Args:
-            md_file_path: location of the markdown file to convert
+            md_file_path: location of md file to convert
```

Comment:
Could we use the full English, rather than abbreviations in docstrings? Makes them much easier to read.